README.md

```diff
@@ -60,6 +62,17 @@ V is being constantly updated. To update V, simply run:
 v up
 ```
 
+2. If you do not have Git installed
```

Comment:
The instructions for this case should be how to install git, NOT how to patch manually source files!

Reply:
Another option is to point people to the prebuilt releases (which are older, but are built specifically for people that do not want to track the latest and greatest).